_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3211,49 +3156,8 @@ public static function validate_service_api_key( $key = null, $service = null )
 	 * @param string $key The API key to be validated.
 	 */
 	public static function validate_service_api_key_mapbox( $key ) {
-		$status          = true;
-		$msg             = null;
-		$mapbox_url      = sprintf(
-			'https://api.mapbox.com?%s',
-			$key
-		);
-		$mapbox_response = wp_safe_remote_get( esc_url_raw( $mapbox_url ) );
-		$mapbox_body     = wp_remote_retrieve_body( $mapbox_response );
-		if ( '{"api":"mapbox"}' !== $mapbox_body ) {
-			$status = false;
-			$msg    = esc_html__( 'Can\'t connect to Mapbox', 'jetpack' );
-			return array(
-				'status'        => $status,
-				'error_message' => $msg,
-			);
-		}
-		$mapbox_geocode_url      = esc_url_raw(
-			sprintf(
-				'https://api.mapbox.com/geocoding/v5/mapbox.places/%s.json?access_token=%s',
-				'1+broadway+new+york+ny+usa',
-				$key
-			)
-		);
-		$mapbox_geocode_response = wp_safe_remote_get( esc_url_raw( $mapbox_geocode_url ) );
-		$mapbox_geocode_body     = wp_remote_retrieve_body( $mapbox_geocode_response );
-		$mapbox_geocode_json     = json_decode( $mapbox_geocode_body );
-		if ( isset( $mapbox_geocode_json->message ) && ! isset( $mapbox_geocode_json->query ) ) {
-			$status = false;
-			$msg    = $mapbox_geocode_json->message;
-		}
-		return array(
-			'status'        => $status,
-			'error_message' => $msg,
-		);
-	}
+		return WPCOM_REST_API_V2_Endpoint_Service_API_Keys::validate_service_api_key_mapbox( $key );
 
-	/**
-	 * Create site option key for service
-	 *
-	 * @param string $service The service  to create key for.
-	 */
-	private static function key_for_api_service( $service ) {
-		return $service . '_api_key';
 	}
```

Comment:
I think we can omit this static method altogether since it's not directly used by this class anymore